app/src/main/java/io/token/sample/Application.java

```diff
@@ -78,11 +78,11 @@ public static void main(String[] args) throws IOException {
 
         // Endpoint for transfer payment, called by client side after user approves payment.
         Spark.get("/fetch-balances", (req, res) -> {
-            //String tokenId = req.queryMap("tokenId").value();
-            String callbackUri = req.raw().getRequestURL().toString()
-                    + "?"
-                    + req.raw().getQueryString();
-            String tokenId = tokenIO.parseTokenRequestCallbackUrl(callbackUri).getTokenId();
+            String tokenId = req.queryMap("tokenId").value();
+            // String callbackUri = req.raw().getRequestURL().toString()
```

Comment:
do we still need this?

Reply:
merchant-sample doesn't use it anymore, so I think is safe to remove